modules/ROOT/pages/installing-multiple-instances.adoc

```diff
@@ -0,0 +1,228 @@
+= Installing Multiple Instances of Runtime Fabric on a Single Cluster
+
+Installing multiple instances of Anypoint Runtime Fabric enables you to share the same cluster among multiple Runtime Fabrics, which helps you to use resources efficiently.
+
+[NOTE]
+To use this feature, you must upgrade to the minimum version of the agent that supports multiple instances.
+
+There will be 1:1 mapping for the agent namespace to the app namespace. Sharing app namespaces is not supported among multiple agent installations.
```

Comment:
```suggestion
There will be 1:1 mapping from the agent namespace to the app namespace. Sharing app namespaces is not supported among multiple agent installations.
```

Reply:
Another clarification we should consider adding is, `application namespaces cannot be shared amongst multiple instances of Runtime Fabric on the same cluster.`